src/diagnostics.md

```diff
@@ -357,14 +357,11 @@ string; this was introduced so that UI tests could both make use of
 the structured JSON and see the "human" output (well, _sans_ colors)
 without having to compile everything twice.
 
-The JSON emitter currently lives in libsyntax/json.rs. (But arguably
-it should live in librustc_errors along with the "human" emitter? It's
-not obvious to the present author why it wasn't moved from libsyntax
-to librustc_errors at the same [time the "human" emitter was
-moved](https://github.com/rust-lang/rust/commit/6ae3502134).)
+The "human" readable and the json format emitter can be found under
+librustc_errors, both where moved from the libsyntax crate.
```

Comment:
where -> were

Also would be nice if you could make it a link to the rustdocs: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_errors/index.html

Reply:
Thanks, I'll change that